Mathlib/Topology/Algebra/Module/Equiv.lean

```diff
@@ -567,6 +567,41 @@ def arrowCongrEquiv (e₁₂ : M₁ ≃SL[σ₁₂] M₂) (e₄₃ : M₄ ≃SL[
     ContinuousLinearMap.ext fun x => by
       simp only [ContinuousLinearMap.comp_apply, apply_symm_apply, coe_coe]
 
+section Pi
+
+/-- Combine a family of linear equivalences into a linear equivalence of `pi`-types.
+This is `Equiv.piCongrLeft` as a `ContinuousLinearEquiv` -/
+def piCongrLeft (R : Type*) [Semiring R] {ι ι' : Type*}
+    (φ : ι → Type*) [∀ i, AddCommMonoid (φ i)] [∀ i, Module R (φ i)]
+    [∀ i, TopologicalSpace (φ i)]
+    (e : ι' ≃ ι) : ((i' : ι') → φ (e i')) ≃L[R] (i : ι) → φ i where
+  __ := Homeomorph.piCongrLeft e
+  __ := LinearEquiv.piCongrLeft R φ e
+
+/-- The space of functions from `S ⊕ T` into a family of topological modules
+is isomorphic to the product of the functions from `S` and the functions from `T`.
+
+This is `Equiv.sumPiEquivProdPi` as a `ContinuousLinearEquiv`.
+-/
+def sumPiEquivProdPi (R : Type*) [Semiring R] (S T : Type*)
+    (A : S ⊕ T → Type*) [∀ st, AddCommMonoid (A st)] [∀ st, Module R (A st)]
+    [∀ st, TopologicalSpace (A st)] :
+    ((st : S ⊕ T) → A st) ≃L[R] ((s : S) → A (Sum.inl s)) × ((t : T) → A (Sum.inr t)) where
+  __ := LinearEquiv.sumPiEquivProdPi R S T A
+  __ := Homeomorph.sumPiEquivProdPi S T A
+
+/-- The space of functions from `PUnit` into a family of topological modules
+is homeomorphic and isomorphic to the value of this function at `()`.
```

Comment:
"homeomorphic and isomorphic" sounds a bit weird to me, what about "topologically isomorphic" or just "isomorphic" ?

Reply:
I think in functional analysis "topologically isomorphic" means "isomorphic as a topological vector space", but being more explicit never hurts.